wagtail/admin/views/pages/listing.py

```diff
@@ -93,9 +94,17 @@ def index(request, parent_page_id=None):
 
     show_ordering_column = request.GET.get("ordering") == "ord"
 
+    side_panels = PageSidePanels(
+        request,
+        parent_page.specific,
+        preview_enabled=True,
```

Comment:
We don’t want the preview to be visible on listing pages.

```suggestion
        preview_enabled=False,
```